src/xdp-app-info.c

```diff
@@ -67,7 +67,6 @@ typedef struct _XdpAppInfoPrivate
   GAppInfo *gappinfo;
   gboolean supports_opath;
   gboolean has_network;
-  gboolean requires_pid_mapping;
```

Comment:
Maybe I screwed something up when refactoring XdpAppInfo? Might be worth double checking what this was used for...

Reply:
It was never used:

 * First mention: 80c88a457b8de6f188f8568ff4d82841b02924af
 * Struct field added: 642ebb31e731f6e9782ea7a8ad6f7a776ab658b2